packages/bridge/src/vl/animation-controls/AnimationControls.ts

```diff
@@ -97,13 +98,33 @@ export class AnimationControls {
     this._animationWidget.addEventListener('seek', (evt) => {
       this._animation.setProgressPct(evt.detail[0] / 100);
       this._animation.notify();
+      this._animationWidget.progressValue = this._formatProgressValue();
     });
 
     this._layer.on('updated', () => {
       this._animationWidget.progress = this._animation.getProgressPct() * 100;
+      this._animationWidget.progressValue = this._formatProgressValue();
     });
   }
 
+  private _formatProgressValue() {
+    const progressValue = this._animation.getProgressValue();
+
+    if (progressValue instanceof Date) {
+      return this._formatCb ? this._formatCb(progressValue) : progressValue.toISOString();
+    }
+
+    if (this._isVLTimeZoneDate(progressValue)) {
```

Comment:
I think we should extract this functions as they're not specific to this class and may be reused.

Reply:
Actually, after taking a look, they're only being used in the Animation Controls.